chartpress.py

```diff
@@ -1004,6 +966,60 @@ def publish_pages(
     _check_call(["git", "push", "origin", "gh-pages"], cwd=checkout_dir)
 
 
+def _check_base_version(base_version):
+    """Verify that a baseVersion config is valid
+
+    If specified, base version needs to:
+
+    1. be a valid semver prerelease
+    2. sort after the latest tag on the branch
+    """
+
+    # check valid value (baseVersion must be semver prerelease)
+    match = _semver2.fullmatch(base_version)
+    if not match:
+        raise ValueError(
+            f"baseVersion: {base_version} must be a valid semver prerelease (e.g. 1.2.3-0.dev), but doesn't appear to be valid."
+        )
+    base_version_groups = match.groupdict()
+    if not base_version_groups["prerelease"]:
+        raise ValueError(
+            f"baseVersion: {base_version} must be a valid semver prerelease (e.g. 1.2.3-0.dev), but is not a prerelease."
+        )
+
+    def _version_number(groups):
+        """Return comparable semver"""
+
+        return (
+            int(groups["major"]),
+            int(groups["minor"]),
+            int(groups["patch"]),
+        )
+
+    # check ordering with latest tag
+    # do not check on a tagged commit
+    tag, count = _get_latest_tag_and_count()
```

Comment:
I think if someone doesn't update `baseVersion: 2.0.0-0.dev` to `baseVersion: 2.0.0-alpha.1` after tagging `2.0.0-alpha.1`, that would be acceptable.

It would mean the future development versions are ordered before the `-alpha.1` when they should really be looking like `2.0.0-alpha.1.git.n.hash` where we updated to `baseVersion: 2.0.0-alpha.1`.